### ISSUE TITLE
User roles is views

### ISSUE THREAD

**Issue by MartinsZieds** [enniel/adonis-acl#6]
Great work on this, i have a small question though -when user is authenticated, what would be the best way how to check for user roles or permissions in the view? (to hide/show some elements). User roles and permissions isn't attached to the `{{ auth.user }}`

**Comment by enniel** [comment]
I created new tags (is, can, scope) for this. You can see examples in [test file](https://github.com/enniel/adonis-acl/blob/master/test/unit/view.spec.js). Use version 1.0.3 or higher

**Comment by MartinsZieds** [comment]
Tnx, I will try it

**Comment by MartinsZieds** [comment]
I sorry to ask, but i'm trying to use the new functionality (i got the newest version)
but when i try to use is it in my views, the `@is` for some reason it doesn't do what excepted 
if this is my view:
```
@layout('main')

@section('content')
<h1>Home</h1>
<p>Welcome to the AdonisJs 4.0</p>
@is('administrator')
  <h2> Hello admin </h2>
@endis
@loggedIn
  <h2> Hello {{ auth.user.id }} </h2>
@else
  <p> Please login </p>
@endloggedIn
@endsection
```
and a user with a user role (slug) `client` opens this view, then he can still see the `Hello admin`

maybe there is something else i need to add in order to make this work?

**Comment by enniel** [comment]
I fixed that in version 1.1.0. You must add Adonis/Acl/Init in globalMiddleware. See more in demo app: https://github.com/enniel/adonis-acl-blog-demo

**Comment by MartinsZieds** [comment]
awesome work, i'll check it out
